pkgs/development/python-modules/swift/default.nix

```diff
@@ -63,21 +63,19 @@ buildPythonPackage rec {
   nativeCheckInputs = [
     boto3
     mock
-    stestr
+    stestrCheckHook
     swiftclient
   ];
 
   # a lot of tests currently fail while establishing a connection
   doCheck = false;
 
-  checkPhase = ''
+  preCheckPhase = ''
```

Comment:
Would preCheck work, too?

Reply:
preCheck would work too, i can change from preCheckPhase to preCheck if you want.

Reply:
I would intuively use preCheck. 😅 If you don't mind, it would be nice to change that. I am at this point mainly waiting for someone to review the hook because I am not suprt confident that it is completely correct.